src/player.js

```diff
@@ -0,0 +1,178 @@
+const Gameboard = require ('./gameboard');
+
+function Player()  {
+
+  return {
+    turn: false,
+    gameEnvironment: Gameboard(),
+    placeShip: function(n, x, y){
+    const newShip =  this.gameEnvironment.createShip(n);
+      this.gameEnvironment.placeShip(newShip, x, y);
+    },
+    printBoard: function(){
```

Comment:
this class should probably just create a player object and returns it. The rest of the functionality you added to it looks like something you should transfer to a file/module of it's own ...which is a DOM MODULE.

You mixed logic and DOM manipulation inside here